src/SepaController.php

```diff
@@ -38,4 +41,28 @@ public function customerDataForm() {
     return new SepaCustomerDataForm();
   }
 
+  /**
+   * Get a form for configuring the payment method.
+   *
+   * @return SepaConfigurationForm
+   *   A new SEPA configuration form.
+   */
+  public function configurationForm() {
+    return new SepaConfigurationForm();
+  }
+
+  /**
+   * Load the intent object and populate the $payment object accordingly.
+   */
+  protected function fetchIntent(\Payment $payment, Api $api) {
+    $intent = parent::fetchIntent($payment, $api);
+    $payment_method = $api->retrievePaymentMethod($intent['payment_method']);
+    $mandate = $api->retrieveMandate($intent['mandate']);
```

Comment:
Those 3 consecutive API calls could be put into a singe call by expanding the SetupIntent object:
```
SetupIntent::retrieve([
  'id' => $id
  'expand' => ['payment_method', 'mandate'],
]);
```

Reply:
Will look into this. Thanks!